### PR TITLE
Added frontend for project and implemented survey creation

### PR DESCRIPTION
**Pull request by RahulAnilkumar8**
Integrated Svelte project and created home page, about page and survey creation. Additionally added get and displaying of created surveys.

### CODE REVIEW

File: frontend/src/components/QuestionCreator.svelte [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<!--        <input type=radio bind:group={answerType} value={"MULTIPLE_CHOICE"}>-->
+<!--        Multiple Choice-->
+<!--    </label>-->
+<!--    <label>-->
+<!--        <input type=radio bind:group={answerType} value={"TEXT"}>-->
+<!--        Text-->
+<!--    </label>-->
+<!--</div>-->
+
+<div class="mcQuestion">
```
Comment (Christopher-Wang) [inline frontend/src/components/QuestionCreator.svelte:27]: This file will be opverwritten by my commits but in the future we should refrain from merging this kind of leftover code to master

Comment (RahulAnilkumar8) [reply frontend/src/components/QuestionCreator.svelte:27]: I 100% agree. I will remove the commented code and update the branch.

Comment (RahulAnilkumar8) [reply frontend/src/components/QuestionCreator.svelte:10]: @Christopher-Wang The branch should be updated to remove the commented out code.

File: frontend/src/components/SurveyList.svelte [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<script>
+    import {onMount} from 'svelte';
+    import Box from "./Box.svelte";
+
+    const url = 'http://localhost:8080/survey/surveyList';
```
Comment (ChristopheTran) [inline frontend/src/components/SurveyList.svelte:5]: Now that the backend is deployed on Heroku, we should avoid using the localhost url.

File: frontend/src/routes/blog/_posts.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+			<p>We're so glad you asked! Come on over to the <a href='https://github.com/sveltejs/svelte'>Svelte</a> and <a href='https://github.com/sveltejs/sapper'>Sapper</a> repos, and join us in the <a href='https://svelte.dev/chat'>Discord chatroom</a>. Everyone is welcome, especially you!</p>
+		`
+	}
+];
+
+posts.forEach(post => {
+	post.html = post.html.replace(/^\t{3}/gm, '');
+});
+
+export default posts;
```
Comment (ChristopheTran) [inline frontend/src/routes/blog/_posts.js:92]: Is this file needed?

File: src/main/java/com/studymonkey/surveychimp/controllers/AnswerController.java [hunk 4]
```diff
@@ -10,7 +10,7 @@
 import javax.annotation.Resource;
 import java.util.List;
 import java.util.Map;
-
+@CrossOrigin(origins = "http://localhost:3000", maxAge = 3600)
```
Comment (ChristopheTran) [inline src/main/java/com/studymonkey/surveychimp/controllers/AnswerController.java:13]: You can remove CORS for localhost now, since we just added a PR to address this.

File: frontend/src/routes/blog/_posts.js (hunk 3 above)
Comment (RahulAnilkumar8) [reply frontend/src/routes/blog/_posts.js:92]: So I left this folder in the project primarily because it had some good examples of how to use some of the svelte features. But I will remove it as its not really needed in the actual project.

**Review by ChristopheTran — APPROVED** [review]

File: frontend/src/components/SurveyList.svelte (hunk 2 above)
Comment (RahulAnilkumar8) [reply frontend/src/components/SurveyList.svelte:5]: I am think that for now I will leave it as localhost. We will address this when we figure out the Travis CI related build configuration for Svelte or when the part Chris is working on is completed.
